### ISSUE TITLE
Configurable length limits for content

### ISSUE THREAD

**Issue by cellio** [codidact/qpixel#582]
Inspired by: 
https://meta.codidact.com/posts/282950  
https://meta.codidact.com/posts/282973  

We currently hard-code the following length limits:

- post title: 1000 characters
- post body: 30,000 characters
- comment: 500 characters
- (comment thread title? unknown)

There are some questions/comments in those meta posts about whether limits are, or could be, configurable per community.  It seems reasonable to me that a community might want articles to be able to be longer, and maybe now that comments are threaded and thus out of the way, some might want to increase the limit there to avoid the "1/3" pattern.  I don't know if there's a use case for decreasing any of these limits (though one person on meta thinks 1000 characters is too long for titles, so who knows?).

Can we make these length limits configurable per community?  (Per category seems too complex without a strong use case.)  The first two could be part of the configuration of the post type; comment limits would need to go somewhere in site settings.

**Comment by Istiakshovon** [comment]
>post title: 1000 characters
No! The limit is 250 character. Tested [here](https://a.dev.codidact.org/posts/174). Tested with 300 characters also. I am looking at db now(haven't find anything), will edit the comment if I find

**Comment by cellio** [comment]
Related (minimum values):

https://github.com/codidact/qpixel/issues/1092 
https://github.com/codidact/qpixel/issues/1103
